traffic_portal/test/integration/PageObjects/ParametersPage.po.ts

```diff
@@ -51,6 +51,7 @@ export class ParametersPage extends BasePage {
   private btnDelete = element(by.buttonText('Delete'));
   private btnYes = element(by.buttonText('Yes'));
   private txtConfirmName = element(by.name('confirmWithNameInput'));
+  private btnTableColumn = element(by.className("caret"))
   private randomize = randomize;
 
   async OpenParametersPage() {
```

Comment:
Same comment for https://github.com/apache/trafficcontrol/blob/df56f95943e45e0355ac4537a67dadf8c7a9e9cf/traffic_portal/test/integration/PageObjects/ParametersPage.po.ts#L62

Reply:
variable `snp` should be constant since it is never re-assigned, 
https://github.com/apache/trafficcontrol/blob/df56f95943e45e0355ac4537a67dadf8c7a9e9cf/traffic_portal/test/integration/PageObjects/ParametersPage.po.ts#L58
https://github.com/apache/trafficcontrol/blob/df56f95943e45e0355ac4537a67dadf8c7a9e9cf/traffic_portal/test/integration/PageObjects/ParametersPage.po.ts#L63